geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/geotools/GeometryUtils.scala

```diff
@@ -105,4 +105,41 @@ object GeometryUtils {
   // checks that there aren't any angled lines
   private def allRightAngles(p: Polygon): Boolean =
     p.getCoordinates.sliding(2).forall { case Array(left, right) => left.x == right.x || left.y == right.y }
+
+  /**
+    * This function checks if a segment crosses the IDL.
+    * @param point1 The first point in the segment
+    * @param point2 The second point in the segment
+    * @return boolean true if the segment crosses the IDL, otherwise false
+    */
+  def crossesIDL(point1:Coordinate, point2:Coordinate): Boolean = {
```

Comment:
Ok, last comment.  We need specific tests for these methods in `GeometryUtilsTest`.